docker-compose.yml

```diff
@@ -13,23 +13,21 @@ services:
     volumes:
       - UserData:/server/UserData
   db:
-    image: mariadb
+    image: postgres
     restart: always
     ports:
-      - "127.0.0.1:3306:3306"
+      - "127.0.0.1:5438:5432"
     volumes:
-      - MariaDBData:/var/lib/mysql
+      - POSTGRESData:/var/lib/postgresql/data 
     environment:
-      MARIADB_ROOT_PASSWORD: CHANGEME
-      MARIADB_USER: evosc
-      MARIADB_PASSWORD: evosc123!
-      MARIADB_DATABASE: evosc
-      MARIADB_AUTO_UPGRADE: always
+      - POSTGRES_USER=evosc
+      - POSTGRES_PASSWORD=evosc123!
+      - POSTGRES_DB=evosc
   adminer:
```

Comment:
Suggestion: Replace adminer with pgAdmin
https://hub.docker.com/r/dpage/pgadmin4/

Reply:
adminer is replaced
when merged, i'll add a short how to connect to db to the dev docs